tornado/test/runtests.py

```diff
@@ -19,6 +19,11 @@
 except NameError:
     from functools import reduce  # py3
 
+try:
+    ResourceWarning
+except NameError:
+    ResourceWarning = None
```

Comment:
this is probably the second-most-interesting change (because python2 does not have ResourceWarning)